charts/k8sgpt/values.yaml

```diff
@@ -14,7 +14,9 @@ deployment:
     requests:
       cpu: "0.2"
       memory: "156Mi"
-
+  securityContext:
```

Comment:
It seems better not to have a default value for securityContext.

Reply:
I have changed to values.yaml file accordingly. No, there is a possibility to set the securityContext but it's not mandatory. Any thoughts on that?